hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -76,6 +77,12 @@ const (
 	TokenSecretTokenKey                       = "token"
 	TokenSecretConfigKey                      = "config"
 	TokenSecretAnnotation                     = "hypershift.openshift.io/ignition-config"
+
+	nodePoolAnnotationCurrentTunedConfig        = "hypershift.openshift.io/nodePoolCurrentTunedConfig"
+	nodePoolAnnotationCurrentTunedConfigVersion = "hypershift.openshift.io/nodePoolCurrentTunedConfigVersion"
+	nodePoolAnnotationTargetTunedConfigVersion  = "hypershift.openshift.io/nodePoolTargetTunedConfigVersion"
```

Comment:
nodePoolAnnotationTargetTunedConfigVersion seems unused

Reply:
Good catch. This should now be fixed.